namespaces/live-1.cloud-platform.service.justice.gov.uk/ppud-replacement-dev/resources/rds.tf

```diff
@@ -13,8 +13,8 @@ module "ppud_replacement_dev_rds" {
   rds_name          = "ppud-replacement-dev"
   rds_family        = "postgres12"
   db_engine         = "postgres"
-  db_engine_version = "13"
-  db_instance_class = "db.t2.small"
+  db_engine_version = "13.2"
```

Comment:
@dazoakley , can u amend to have just the major version. We have auto_minor_upgrade enabled and terraform complaints for this change when AWS does does the automatic minor upgrades.